### PR TITLE
add rotation and vibration options to interface

### PR DESCRIPTION
**Pull request by jtravs**
This is a small interface enhancement to allow selection of which Raman contributions to include.

### CODE REVIEW

File: src/Interface.jl [hunk 1]
```diff
@@ -490,10 +495,11 @@ function makeresponse(grid::Grid.RealGrid, gas, raman, kerr, plasma, thg, pol)
     end
     makeplasma!(out, grid, gas, plasma, pol)
     if raman
+        rr = Raman.raman_response(grid.to, gas, rotation=rotation, vibration=vibration)
```
Comment (chrisbrahms) [inline src/Interface.jl:498]:
```suggestion
        rr = Raman.raman_response(grid.to, gas; rotation, vibration)
```

File: src/Interface.jl [hunk 2]
```diff
@@ -530,7 +537,10 @@ function makeresponse(grid::Grid.EnvGrid, gas, raman, kerr, plasma, thg, pol)
             push!(out, Nonlinear.Kerr_env(PhysData.γ3_gas(gas)))
         end
     end
-    raman && push!(out, Nonlinear.RamanPolarEnv(grid.to, Raman.raman_response(grid.to, gas)))
+    if raman
+        rr = Raman.raman_response(grid.to, gas, rotation=rotation, vibration=vibration)
```
Comment (chrisbrahms) [inline src/Interface.jl:541]:
```suggestion
        rr = Raman.raman_response(grid.to, gas; rotation, vibration)
```

**Review by chrisbrahms — APPROVED** [review]
LGTM 
Small suggestions on using most concise syntax.

File: src/Interface.jl (hunk 1 above)
Comment (jtravs) [reply src/Interface.jl:498]: Doesn't this syntax only work on 1.7?

Comment (jtravs) [reply src/Interface.jl:498]: Never mind, I see it was added in 1.5

**Pull request merged**